example/metric/go.mod

```diff
@@ -6,6 +6,7 @@ replace github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metr
 
 require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric v0.2.1
-	go.opentelemetry.io/otel v0.9.0
+	go.opentelemetry.io/otel v0.10.0
+	go.opentelemetry.io/otel/sdk v0.10.0
```

Comment:
How come this got added? Seems weird given that no code changed (likewise for elsewhere)

Reply:
Nvm just saw https://github.com/open-telemetry/opentelemetry-go/pull/985